.changeset/witty-foxes-punch.md

```diff
@@ -0,0 +1,5 @@
+---
+'modular-scripts': patch
```

Comment:
I'd argue this is probably more of a minor than a patch. 🛩️

Reply:
I'm using minors before 1.0 to mean breaking changes, think it would still apply?

(I don't feel super strongly about it, so either works for me.)